File: python/2.2.6/2.2.6_MPU6050.py
#!/usr/bin/env python3

import MPU6050 
import time

mpu = MPU6050.MPU6050()     #instantiate a MPU6050 class object
ac = [0]*3               #store accelerometer data
gy = [0]*3                #store gyroscope data

def setup():
    mpu.dmp_initialize()    #initialize MPU6050
    
def loop():
    while(True):
        ac = mpu.get_acceleration()      #get accelerometer data
        gy = mpu.get_rotation()           #get gyroscope data
        print("Acceleration:\n")
        print("X:%.2f g  Y:%.2f g  Z:%.2f g\n"%(ac[0]/16384.0,ac[1]/16384.0,ac[2]/16384.0))
        print("Angular velocity:\n")
        print("X:%.2f d/s Y:%.2f d/s Z:%.2f d/s\n"%(gy[0]/131.0,gy[1]/131.0,gy[2]/131.0))
        time.sleep(0.1)
        
if __name__ == '__main__':     # Program start from here
    setup()
    try:
        loop()
    except KeyboardInterrupt:  # When 'Ctrl+C' is pressed,the program will exit.
        pass

File: c/2.2.6/2.2.6_MPU6050.cpp
/**********************************************************************
* Filename    : 2.2.6_MPU6050.cpp
* Description : Read the Raw data of MPU6050
* Author      : Robot
* E-mail      : support@sunfounder.com
* website     : www.sunfounder.com
* Update      : Daisy    2019/08/01
**********************************************************************/
#include <stdio.h>
#include <stdint.h>
#include <unistd.h>
#include "I2Cdev.h"
#include "MPU6050.h"

MPU6050 mpu;      //instantiate a MPU6050 class object

int16_t ax, ay, az;     //store acceleration data
int16_t gx, gy, gz;     //store gyroscope data

void loop() {
    // read raw accel/gyro measurements from device
    mpu.getMotion6(&ax, &ay, &az, &gx, &gy, &gz);
    // display accel/gyro x/y/z values
    printf("Acceleration:\n");
    printf("X:%.2f g  Y:%.2f g  Z:%.2f g\n",(float)ax/16384,(float)ay/16384,(float)az/16384);
    printf("Angular velocity:\n");
    printf("X:%.2f d/s Y:%.2f d/s Z:%.2f d/s\n",(float)gx/131,(float)gy/131,(float)gz/131);
}

int main()
{
    mpu.initialize();     //initialize MPU6050
    // verify connection
    printf(mpu.testConnection() ? "" : "initialize failed\n");
    while(1){
        loop();
    }
    return 0;
}

